.github/workflows/test.yml

```diff
@@ -765,9 +692,18 @@ jobs:
           SQL_DB_HOST: ${{ secrets.SQL_DB_HOST_LOCAL }}
         run: npm run db:migrate
 
+      - name: Project
+        id: project
+        run: |
+          directory=$(echo ${{ matrix.spec }} | cut -d / -f2)
+          echo "::set-output name=directory::$directory"
+
       - name: Execute
         working-directory: ./e2e-tests
         env:
           SQL_DB_HOST: ${{ secrets.SQL_DB_HOST_LOCAL }}
           TZ: ${{ vars.TIMEZONE }}
-        run: npx cypress run --config video=false,screenshotOnRunFailure=false --project ./ukef --record false --spec "./ukef/cypress/e2e/journeys/portal/${{ matrix.spec }}"
+          # Various feature flags
+          FF_TFM_FACILITY_END_DATE_ENABLED: 'false'
```

Comment:
I'm not up to speed with the feature flags in DTFS - since this flow is called "E2E Feature flag", should this flag be enabled?

Is there more 1x feature flag? If so i'd expect them all to be listed and `true` here

Reply:
Good point, @MarRobSoftwire I have refactored the `test.yml` to keep all FF E2E tests together I copied `FF_TFM_FACILITY_END_DATE_ENABLED` and `GEF_DEAL_VERSION` env variables and they both are falsey, should they be truthy for FF E2E to be tested effectively?